RedHatInsights/patchman-ui
# chore: Collect coverage from component tests

This combines coverage reports from jest and cypress together before commiting them to codecov.

# Description

Associated Jira ticket: https://issues.redhat.com/browse/SPM-1826

# How to test the PR

Run tests locally, check that two coverage folders are generated, and then try  to run `npm run coverage`.

# Before the change


# After the change


# Dependent work link


# Checklist:
